src/api/make.js

```diff
@@ -62,8 +62,10 @@ export default async (providedOptions = {}) => {
     forgeConfig = await getForgeConfig(dir);
   });
 
-  if (!['darwin', 'win32', 'linux'].includes(platform)) {
-    throw new Error(`'${platform}' is an invalid platform. Choices are 'darwin', 'win32' or 'linux'`);
+  const actualTargetPlatform = platform;
+  platform = platform === 'mas' ? 'darwin' : platform;
```

Comment:
I don't think this is what you want. As far as I can tell this results in `out/App-darwin-x64`